### DISCUSSION TITLE
Location request on HTTP fail the whole app converstion

### DISCUSSION THREAD

**Discussion by sala91** [open-webui/open-webui#4133] [General]
# Bug Report

## Description
When you enable location request in profile settings and app is running outside localhost on http, then webui stops responding to requests. When you disable location requests, the webui starts responding to requests again. Imo, having denied location requests, should not be the bases to stop chat from working.

## Environment

- **Open WebUI Version:** v0.3.10

- **Operating System:** Windows 11
- **Browser (if applicable):** Safari 17.5 Mac 

## Reproduction Details

**Confirmation:**

- [ X ] I have read and followed all the instructions provided in the README.md.
- [ X ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Installation Method

Classic docker install without GPU support

**Comment by Aliang-code** [comment]
Is there any progress?
I can't find any explanation as to why it needs to request the user's location?
it worries me.
